### ISSUE TITLE
The hyper-parameters w_s and w_t setting in MDD

### ISSUE THREAD

**Issue by jingzhengli** [thuml/Transfer-Learning-Library#57]
Hi, as for algorithm MDD, in the class MarginDisparityDiscrepancy(nn.Module):    what are the w_s and w_t, and how to set them?

 def forward(self, y_s: torch.Tensor, y_s_adv: torch.Tensor, y_t: torch.Tensor, y_t_adv: torch.Tensor,
                w_s: Optional[torch.Tensor] = None, w_t: Optional[torch.Tensor] = None) -> torch.Tensor:

        source_loss = -self.margin * self.source_disparity(y_s, y_s_adv)
        target_loss = self.target_disparity(y_t, y_t_adv)
        **if w_s is None:**
            w_s = torch.ones_like(source_loss)
        source_loss = source_loss * w_s
        **if w_t is None:**
            w_t = torch.ones_like(target_loss)
        target_loss = target_loss * w_t

        loss = source_loss + target_loss
        if self.reduction == 'mean':
            loss = loss.mean()
        elif self.reduction == 'sum':
            loss = loss.sum()
        return loss

**Comment by JunguangJiang** [comment]
You can set them to None in the standard domain adaptation setting.
In partial DA or open-set DA, you might need to set instance-wise weight.

**Comment by jingzhengli** [comment]
GOT IT. thanks for the quick reply.
